projects/packages/my-jetpack/src/products/class-boost.php

```diff
@@ -98,7 +98,7 @@ public static function get_description() {
 	 * @return string
 	 */
 	public static function get_long_description() {
-		return __( 'Jetpack Boost gives your site the same performance advantages as the world’s leading websites, no developer required.', 'jetpack-my-jetpack' );
+		return __( 'Fast sites get more page visits, more conversions and better SEO rankings. Boost speeds up your site in seconds.', 'jetpack-my-jetpack' );
```

Comment:
Nitpick
```suggestion
		return __( 'Fast sites get more page visits, more conversions, and better SEO rankings. Boost speeds up your site in seconds.', 'jetpack-my-jetpack' );
```
Also, should we say "Jetpack Boost" here instead of just "Boost"?